hydra/test_utils/launcher_common_tests.py

```diff
@@ -252,11 +253,13 @@ def sweep_2_jobs(sweep_runner: TSweepRunner, overrides: List[str]) -> None:
             assert job_ret.hydra_cfg.hydra.job.name == "a_module", (
                 "Unexpected job name: " + job_ret.hydra_cfg.hydra.job.name
             )
-            verify_dir_outputs(job_ret, job_ret.overrides)
-            path = temp_dir / str(i)
-            assert path.exists(), "'{}' does not exist, dirs: {}".format(
-                path, [x for x in temp_dir.iterdir() if x.is_dir()]
-            )
+            verify_dir_outputs(job_ret, sweep.remote, job_ret.overrides)
+            if not sweep.remote:
+                temp_dir = Path(sweep.temp_dir)
+                path = temp_dir / str(i)
+                assert path.exists(), "'{}' does not exist, dirs: {}".format(
+                    path, [x for x in temp_dir.iterdir() if x.is_dir()]
+                )
```

Comment:
so essentially you are not validating the output directory if it's remote?
can't say I like it, I would rather that you rsynced the dirs locally and tested them.